packages/react-components/components/input/src/textarea.jsx

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable react/forbid-foreign-prop-types */
+
+import { Ref, TextArea as SemanticTextArea } from "semantic-ui-react";
+import { bool, func, number, object, oneOfType } from "prop-types";
+import { forwardRef, useEffect, useImperativeHandle, useRef } from "react";
+import { isNil } from "lodash";
+import { mergeClasses, throwWhenUnsupportedPropIsProvided } from "@orbit-ui/react-components-shared";
+
+const UNSUPPORTED_PROPS = [];
+
+const propTypes = {
+    /**
+     * Whether or not the textarea should autofocus on render.
+     */
+    autofocus: bool,
+    /**
+     * Delay before trying to autofocus.
+     */
+    autofocusDelay: number,
+    /**
+     * Whether or not the textarea is invalid.
+     */
+    error: bool,
+    /**
+     * Whether or not the textarea is valid.
+     */
+    success: bool,
+    /**
+     * Whether or not a user is currently interacting with the textarea.
+     */
+    focused: bool,
+    /**
+     * Whether or not the textarea is transparent.
+     */
+    transparent: bool,
+    /**
+     * @ignore
+     */
+    forwardedRef: oneOfType([object, func])
+};
+
+const defaultProps = {
+    autofocus: false,
+    error: false,
+    success: false,
+    focused: false,
+    transparent: false
+};
+
+function getTextAreaElement(textAreaRef) {
+    return textAreaRef.current;
+}
+
+function focus(textAreaRef) {
+    if (!isNil(textAreaRef.current)) {
+        getTextAreaElement(textAreaRef).focus();
```

Comment:
dont need to call getTextAreaElement, you already have the ref element that you need to focus